src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -113,6 +113,17 @@ public Http2Stream(HttpRequestMessage request, Http2Connection connection, int s
                     _requestBodyCancellationSource = new CancellationTokenSource();
                 }
 
+                if (_request.HasHeaders && _request.Headers.ExpectContinue == true)
+                {
+                    // Create a TCS that will complete when one of two things occurs:
```

Comment:
This comment is now duplicated here and in WaitFor100ContinueAsync.

I think it would be better to 
(a) have a short comment here, something like "Create a TCS for handling Expect: 100-continue semantics. See WaitFor100ContinueAsync. Note we need to create this in the constructor, because we can receive a 100 Continue response at any time after the constructor finishes."
(b) Update the comment in WaitFor100Continue to reflect these changes (i.e. don't say "Create a TCS" since we're not actually creating it there anymore)

Reply:
updated.